main/http_status_codes.h

```diff
@@ -22,22 +22,30 @@ typedef struct _http_response_status_code_pair {
 	const char *str;
 } http_response_status_code_pair;
 
+/* https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml */
+
 static const http_response_status_code_pair http_status_map[] = {
 	{ 100, "Continue" },
 	{ 101, "Switching Protocols" },
+	{ 102, "Processing" },
+	{ 103, "Early Hints" },
 	{ 200, "OK" },
 	{ 201, "Created" },
 	{ 202, "Accepted" },
 	{ 203, "Non-Authoritative Information" },
 	{ 204, "No Content" },
 	{ 205, "Reset Content" },
 	{ 206, "Partial Content" },
+	{ 207, "Multi-Status" },
+	{ 208, "Already Reported" },
+	{ 226, "IM Used" },
 	{ 300, "Multiple Choices" },
 	{ 301, "Moved Permanently" },
 	{ 302, "Found" },
 	{ 303, "See Other" },
 	{ 304, "Not Modified" },
 	{ 305, "Use Proxy" },
+	{ 306, "(Unused)" },
```

Comment:
We should probably not add unused status codes in this map, otherwise `get_status_string()` will actually return `(Unused)`.

Reply:
what shall we do then ? these codes are assigned and valid.. yet no longer have a definition.

Reply:
If I understand correctly, these status codes are deprecated by this standard and no longer have a definition. The standard only recognizes they add a definition in the past, and reserves them so they can not be assigned in the future, which could break existing code.

So they are not standard status codes, and are only listed for the purpose of not assigning or reusing them in the future.

I think we should either use the old description instead of `(Unused)` (`Switch Proxy` and `I'm a teapot`), or omit these codes entirely from this list. In the latter case, applications can still use these codes, but the cli server will assume the default description of `Unknown Status Code`, and the CGI SAPI will not provide a description.